### PR TITLE
FISH-9146 FISH-9147 Upgrade Jakarta Pages and WASP

### PR DESCRIPTION
**Pull request by kalinchan**

### CODE REVIEW

File: appserver/packager/external/jakarta-ee11-shim/pom.xml [hunk 1]
```diff
@@ -91,6 +91,13 @@
             <artifactId>jakarta.ws.rs-api</artifactId>
             <scope>provided</scope>
         </dependency>
+
+        <!-- Downgrade shim for JSP, required by WASP. Remove once WASP upgraded -->
```
Comment (Pandrex247) [inline appserver/packager/external/jakarta-ee11-shim/pom.xml:95]:
```suggestion
        <!-- Downgrade shim for JSTL, required by WASP. Remove once JSTL-API upgraded -->
```

File: appserver/packager/external/jakarta-ee11-shim/pom.xml [hunk 2]
```diff
@@ -124,7 +131,9 @@
                                     <!-- Downgrade shim for Jakarta Authorization, required by Soteria 3.0 -->
                                     jakarta.security.jacc-api,
                                     <!-- Downgrade shim for JAX-RS, required by Soteria. Remove once Soteria upgraded -->
-                                    jakarta.ws.rs-api
+                                    jakarta.ws.rs-api,
+                                    <!-- Downgrade shim for JSP, required by WASP. Remove once WASP upgraded -->
```
Comment (Pandrex247) [inline appserver/packager/external/jakarta-ee11-shim/pom.xml:135]:
```suggestion
                                    <!-- Downgrade shim for JSTL, required by WASP. Remove once JSTL API upgraded -->
```

File: appserver/packager/external/jakarta-ee11-shim/pom.xml [hunk 3]
```diff
@@ -227,7 +236,12 @@
                                     jakarta.ws.rs.container;
                                     jakarta.ws.rs.core;
                                     jakarta.ws.rs.ext;
-                                    jakarta.ws.rs.sse;version="3.1.99";shim=true
+                                    jakarta.ws.rs.sse;version="3.1.99";shim=true,
+
+                                    <!-- Downgrade shim for JSP, required by WASP. Remove once WASP upgraded -->
```
Comment (Pandrex247) [inline appserver/packager/external/jakarta-ee11-shim/pom.xml:241]:
```suggestion
                                    <!-- Downgrade shim for JSTL, required by WASP. Remove once JSTL API upgraded -->
```

**Review by Pandrex247 — CHANGES_REQUESTED** [review]

File: appserver/packager/external/jakarta-ee11-shim/pom.xml (hunk 3 above)
Comment (Pandrex247) [inline appserver/packager/external/jakarta-ee11-shim/pom.xml:241]: This comment isn't accurate - this PR _is_ upgrading WASP.
It's the JSTL API which isn't resolving.

Comment (kalinchan) [reply appserver/packager/external/jakarta-ee11-shim/pom.xml]: That explains why I couldn't remove the shim 😆 Updated.

**Review by Pandrex247 — APPROVED** [review]
Issues [FISH-9571](https://payara.atlassian.net/browse/FISH-9571) & [FISH-9572](https://payara.atlassian.net/browse/FISH-9572) raised to track upgrading JSTL.

**Pull request merged**